libs/components/modals/src/lib/modules/modal/modal.component.html

```diff
@@ -59,7 +59,7 @@
     <div
       class="sky-modal-content sky-padding-even-large"
       role="region"
-      tabindex="0"
+      tabindex="-1"
```

Comment:
I'm not 100% sure about this change, but Level Access dinged us for having this focusable since there is no interaction for the user?